### DISCUSSION TITLE
Is there any way to increase Mob EXP drops?

### DISCUSSION THREAD

**Discussion by teefssi** [RobertSkalko/Age-of-Exile#85] [Q&A]
I've been aimlessly looking at the source  for hours with no knowledge of  data packs, or editing configurations. So I have to resort to this question: How do you increase it? I've tried making my own datapack and ended up crashing my server. If someone could kindly answer, it would save so much time. Thanks in advance!

**Comment by teefssi** [comment]
KubeJs can't solve it either, since aternos doesnt support KubeJs on its mods.

**Comment by RobertSkalko** [comment]
You will have more luck on the discord, just be patient.

**Reply by teefssi** [reply]
Alright! I've asked this question in #help and #general too, my username's teefssi#8780 c:
